apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -106,11 +108,25 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   eventType.schedule = null;
 
+  const locations = eventType.locations as LocationObject[];
+
   const eventTypeObject = Object.assign({}, eventType, {
     metadata: (eventType.metadata || {}) as JSONObject,
     periodStartDate: eventType.periodStartDate?.toString() ?? null,
     periodEndDate: eventType.periodEndDate?.toString() ?? null,
     recurringEvent: (eventType.recurringEvent || {}) as RecurringEvent,
+    locations: locations?.map((el) => {
```

Comment:
Third time repeating this code feel like the need of abstraction. Or filtering way upstream.